File: profile/maven2/demo/src/main/java/com/example/demo/dto/ProfileAdmin.java
package com.example.demo.dto;

import com.example.demo.db.model.*;
import java.util.*;
import java.time.LocalDateTime;

/**
* @class ProfileAdmin
*
*/
public class ProfileAdmin {
    public long id;
    public String firstName;
    public String lastName;
    public String username;
    public String email;
    
    public LocalDateTime createdDate;
    public LocalDateTime lastLoginDate;
    public Address mailingAddress;
    public Address physicalAddress;
    public List<Role> roles;
    public long themeId;
    public boolean active;
}

File: profile/angular/src/app/shared/dto/model/profile-admin.ts
import { Address } from './address';
import { Role } from './role';

/**
* @class ProfileAdmin
* 
*/
export class ProfileAdmin {
    id: number = 0;
    firstName: string = "";
    lastName: string = "";
    username: string = "";
    email: string = "";
    
    createdDate: Date = new Date();
    lastLoginDate: Date = new Date();
    mailingAddress: Address = new Address();
    physicalAddress: Address = new Address();
    roles: Role[] = [];
    themeId: number = 0;
    active: boolean = false;
}

